checkin/models.py

```diff
@@ -11,12 +20,30 @@ def __str__(self):
         return self.title
 
     def locale_title(self, language_code):
+        """Returns skill name in correct language.
+
+        Only 'nb' is recognised, everything else will return the english skill name.
```

Comment:
Should rather be "Supports Norwegian Bokmål and English, defaulting to English for all other language codes."